pkg/registrymirror/registrymirror.go

```diff
@@ -69,6 +78,15 @@ func (r *RegistryMirror) CuratedPackagesMirror() string {
 	return r.NamespacedRegistryMap[constants.DefaultCuratedPackagesRegistryRegex]
 }
 
+// Credentials returns the credential for the registry mirror.
+func (r *RegistryMirror) Credentials() (string, string, error) {
+	username, password, err := config.ReadCredentials()
```

Comment:
Can you move the logic from `config.ReadCredentials()` to here and remove that function?

Reply:
What about other places that call `config.ReadCredentials()`? They now have to get notion of `RegistryMirror` in order to call `Credentials()`. Any thoughts?

Reply:
Can we just make this since it doesn't really need the RegistryMirror object?
It's just reading env and return the values
```
func Credentials() (string, string, error) {
```